draft-demarco-oauth-status-attestations.md

```diff
@@ -260,19 +260,41 @@ POST /status HTTP/1.1
 Host: issuer.example.org
 Content-Type: application/x-www-form-urlencoded
 
-credential_pop=$CredentialPoPJWT
+credential_pop = [$CredentialPoPJWT]
 ~~~
 
```

Comment:
please, the previous example must be moved out from the introduction.

A new section "Status Attestation Request" must be created where the description of the request must be made (moving the text from line 253 in this new section), then the definition of the request parameter, where the credential_pop must be defined.

Please, read other IETF specifications and use them as example about the editorial cut we expect to have.

the text related to the verification of the request should be moved in a specialized section as well.